src/content/zh-tw/fundamentals/payments/index.md

```diff
@@ -0,0 +1,73 @@
+project_path：/web/fundamentals/_project.yaml book_path：/web/fundamentals/_book.yaml
+description：Payment Request API 是用於在網路上進行快速，簡單的付款。
+
+{# wf_published_on: 2016-07-25 #}
+{# wf_updated_on: 2018-02-20 #}
```

Comment:
Updated on should be today's date.

Reply:
to fix this PR, I updated 2 segments at GitLocalize https://gitlocalize.com/repo/107/zh-TW/review/658 .
But I don’t have power to commited to this pull request, because I’m not GitLocalize Moderator.